### ISSUE TITLE
wrong value after use toCurrencyString() with mantissaLength

### ISSUE THREAD

**Issue by carterAtFunNow** [caseyryan/flutter_multi_formatter#61]
```
final test1 = 1000.00;
final test2 = 1000.50;

print(test1.toCurrencyString(mantissaLength: 0));
print(test2.toCurrencyString(mantissaLength: 0));
```

expect: 
```
1,000
1,000

```
actual:
```
10,000
10,005
```

**Comment by caseyryan** [comment]
I'll check it

**Comment by caseyryan** [comment]
Fixed in 2.4.0
